stack/stack_mac.yaml

```diff
@@ -39,7 +39,7 @@ udunits:
 hdf5:
   build: YES
   version: 1.10.6
-  shared: YES
```

Comment:
Same for `stack_mac.yaml`

Reply:
Addressed by fd8237c9a467d55de02a9440c7538096bf806baa